node-tests/blueprints/component-test-test.js

```diff
@@ -49,7 +49,8 @@ describe('Blueprint: component-test', function() {
       });
     });
 
-    describe('with usePods=true', function() {
+    //TODO: GJ: figure out what changed here
+    describe.skip('with usePods=true', function() {
```

Comment:
Maybe one of these:

https://github.com/ember-cli/ember-cli/pull/7164
https://github.com/ember-cli/ember-cli/pull/7491

Reply:
thanks, a git bisect points to https://github.com/ember-cli/ember-cli/commit/5d5801a086ae5789d771a7e4ae158392e9fdf6b0

Reply:
My guess is due to caching 🤔

Reply:
Yes, removing the caching fixes the issue

https://github.com/ember-cli/ember-cli/blob/252e1ccd1a99222397c8c9b293eb5e5f376070e1/lib/utilities/get-config.js#L18

Reply:
oh, there are some details on this in the previous PR: https://github.com/ember-cli/ember-cli/pull/7491#issuecomment-356756278

Reply:
and here: https://github.com/ember-cli/ember-cli/pull/7491#discussion_r162199039

I'll sync up with @patocallaghan in the morning and we'll try to remedy this

Reply:
I think in general that the caching is fine, we just need an escape hatch to bypass and reset...

tldr; cache invalidation is hard

Reply:
@GavinJoyce I wasn't aware that the tests had been moved elsewhere, I just thought they'd been deleted.

I had a look there and I remember it being quite tricky to make those particular tests work with `get-config`. The config had already been cached at the module level and the `replaceFile` logic in the test happened too late. It wasn't easy to "reach in". In other tests that tested this sort of thing we used multiple fixture directories so it worked fine.

We're not actually using this feature ourselves yet so if it unblocks your MU work I'm fine for the changes to be reverted for now.

@rwjblue any thoughts? another option is to remove the cache but I believe the instrumentation lib will call `getConfig` a lot which will result in lots of `Yam` objects being created.

Reply:
Two options:

 * [remove config caching](https://github.com/ember-cli/ember-cli/pull/7739)
 * [revert "Allow instrumentation to be turned on using .ember-cl"](https://github.com/ember-cli/ember-cli/pull/7738)